### PR TITLE
Add map quicklook

### PR DESCRIPTION
**Pull request by nekhtiari**
Use map_quicklooks for visualizing quicklooks.

### CODE REVIEW

File: utils/widgets.py [hunk 1]
```diff
@@ -235,7 +241,9 @@ def show_quicklooks():
                 sensor=self.sensors,
                 output_directory=self.outdir / "quicklooks",
             )
-            self.catalog.plot_quicklooks(figsize=(20, 20))
+            display(
+                self.catalog.map_quicklooks(scenes=self.search_results, aoi=self.aoi)
```
Comment (rodrigoalmeida94) [inline utils/widgets.py:245]: Nice!

**Review by rodrigoalmeida94 — APPROVED** [review]
LGTM! Can you maybe remove all this folders (like `demo2`, `my_new_folder` and `dakar_complex_pleiades_fullscene`), must have checked them in in previous PRs by mistake...

File: utils/widgets.py (hunk 1 above)
Comment (nekhtiari) [reply utils/widgets.py:245]: > LGTM! Can you maybe remove all this folders (like `demo2`, `my_new_folder` and `dakar_complex_pleiades_fullscene`), must have checked them in in previous PRs by mistake...

Done!

**Comment by nekhtiari** [comment]
> LGTM! Can you maybe remove all this folders (like `demo2`, `my_new_folder` and `dakar_complex_pleiades_fullscene`), must have checked them in in previous PRs by mistake...

Done!

**Pull request merged**